bitrise.yml

```diff
@@ -190,6 +191,38 @@ workflows:
         inputs:
         - dir_to_check: $CLONE_INTO_DIR
 
+  _test_checkout_commit_with_other_branch:
+    before_run:
+      - _create_tmpdir
+    steps:
+      - path::./:
+          run_if: true
+          inputs:
+            - clone_into_dir: $CLONE_INTO_DIR
+            - commit: e9667364b397ec5d97b267240928432f74118d72
+            - tag: ""
+            - branch: "master"
+            - pull_request_id: ""
+            - pull_request_merge_branch: ""
+            - pull_request_repository_url: ""
+            - branch_dest: ""
+            - clone_depth: ""
+      - script:
+          inputs:
+            - content: |-
+                #!/bin/bash
```

Comment:
This script step is repeated multiple times. Could we either extract it for reuse now or add it to our list of improvements for the git clone step?